contracts/InvestorVesting.sol

```diff
@@ -125,16 +125,19 @@ contract InvestorVesting is IInvestor, Ownable {
                 _vestingDetails[i].lastClaimedTime == 0,
                 "Last claimed time is not valid"
             );
-            //New beneficiary's initial amount must be greater than 0
-            require(
-                _vestingDetails[i].initialAmount > 0,
-                "Initial release amount is not valid"
-            );
-            //New beneficiary's initial claimed must be false
-            require(
-                _vestingDetails[i].initialClaimed == false,
-                "Initial claimed is not valid"
+            if (_vestingDetails[i].initialAmount > 0) {
+                //New beneficiary's initial claimed must be false
+                require(
+                    _vestingDetails[i].initialClaimed == false,
+                    "Initial claimed is not valid"
+                );
+            } else {
+                //New beneficiary's initial claimed must be true if there is no initialAmount
+                require(
+                    _vestingDetails[i].initialClaimed == true,
+                    "Initial claimed is not valid"
             );
+            }
```

Comment:
This could also be written like this:
```suggestion
            require(
                (_vestingDetails[i].initialAmount > 0 && !_vestingDetails[i].initialClaimed)
                    || (_vestingDetails[i].initialAmount == 0 && _vestingDetails[i].initialClaimed),
                "Initial claimed is not valid"
            );
```